packages/amplify-e2e-tests/src/__tests__/api_2.test.ts

```diff
@@ -129,43 +127,6 @@ describe('amplify add api (GraphQL)', () => {
     expect(updateResultData.updateNote.note).toEqual(updateInput.input.note);
   });
 
-  it('init a project with conflict detection enabled and toggle disable', async () => {
```

Comment:
This case takes the majority time of the whole test suites(around 30~40mins). This is moved to a new test file `api_11`